tedana/decomposition/eigendecomp.py

```diff
@@ -18,6 +18,23 @@
 Z_MAX = 8
 
 
+def run_pca(data, mlepca=True):
+    """
+    Run PCA or SVD.
```

Comment:
This PCA is an SVD, so I find this docstring / function name a little confusing. Maybe we could say "run_svd" instead ?
The difference afaict seems to be that `mle` option automatically estimates the number of components using the method from Thomas P. Minka: Automatic Choice of Dimensionality for PCA. NIPS 2000: 598-604.

Reply:
Done. Do you want me to include the Minka citation with duecredit? It will involve splitting off parts of `run_svd` into a `_run_mlepca` (with the citation) and a `_run_svd` (without). Sort of like what we did with `make_optcom`:
https://github.com/ME-ICA/tedana/blob/cb01d725d05f0102e2da89ad50c32905c68a79fd/tedana/model/combine.py#L153-L156

Reply:
In a perfect world, the citation would be generated by `sklearn`, but I don't see why we can't do it ourselves. Done!